packages/demo/examples/01-xy-chart/HorizontalBarChart.jsx

```diff
@@ -69,51 +62,58 @@ class HorizontalBarChartExample extends React.PureComponent {
   }
 
   render() {
-    const { screenWidth } = this.props;
     const { direction } = this.state;
-    const categoryScale = { type: 'band', paddingInner: 0.15 };
+    const categoryScale = { type: 'band', paddingInner: 0.4 };
     const valueScale = { type: 'linear' };
     const horizontal = direction === 'horizontal';
 
     return (
       <div className="horizontal-bar-demo">
         {this.renderControls()}
-        <XYChart
-          theme={theme}
-          width={Math.min(700, screenWidth / 1.5)}
-          height={Math.min(700 / 2, screenWidth / 1.5 / 2)}
+        <ResponsiveXYChart
           ariaLabel="Required label"
+          eventTrigger="series"
           xScale={horizontal ? valueScale : categoryScale}
           yScale={horizontal ? categoryScale : valueScale}
           margin={{ left: 100, top: 64, bottom: 64 }}
         >
+          <PatternCircles
+            id="horizontal_bar_circles"
+            width={6}
+            height={6}
+            radius={2}
+            fill={allColors.blue[horizontal ? 2 : 8]}
+            strokeWidth={0}
+          />
           <BarSeries
+            fill={allColors.blue[horizontal ? 8 : 2]}
             horizontal={horizontal}
             data={horizontal ? categoryHorizontalData : categoryData}
           />
+          <BarSeries
```

Comment:
I like the circle patterns a lot. nit. Can we actually use `mask` + `pattern` to reduce the duplication of bar series? One example here is https://bl.ocks.org/jfsiii/7772281.  But that means we may need to change @vx for the pattern filling right?

Reply:
^that's a good suggestion. I think we'd need to update how vx does it though, yeah, so maybe in a future PR. probably a good perf improvement.